facades/PC/README.md

```diff
@@ -0,0 +1,102 @@
+                  [--server-port=<serverPort>]
+                  
+      --[no-]crash-report   Enable crash reporting
+      --create-last-game    Recreates the world of the latest game with a new save file on startup
+      -h, /h, -?, /?, -help, /help, --help
+                            Show help
+      --headless            Start headless (no graphics)
+      --homedir=<homeDir>   Path to home directory
+      --load-last-game      Load the latest game on startup
+      --max-data-size=<size>
+                            Set maximum process data size [Linux only]
+      --oom-score=<score>   Adjust out-of-memory score [Linux only]
+      --override-default-config=<overrideConfigPath>
+                            Override default config
+      --permissive-security
+      --[no-]save-games     Enable new save games
+      --server-port=<serverPort>
+                            Change the server port
+      --[no-]sound          Enable sound
+      --[no-]splash         Enable splash screen
+
+For details, see
+ https://github.com/MovingBlocks/Terasology/wiki/Advanced-Options
+
+Alternatively use our standalone Launcher from
+ https://github.com/MovingBlocks/TerasologyLauncher/releases
+```
+
+### Gradle
+
+The PC Facade provides a couple of Gradle tasks for starting the game on the PC.
+These "Terasology run tasks" offer various default configurations, for instance, to enable debugging or start in _headless mode_.
+
+``` 
+Terasology run tasks
+--------------------
+debug - Run 'Terasology' to play the game as a standard PC application (in debug mode)
+game - Run 'Terasology' to play the game as a standard PC application
+permissiveNatives - Run 'Terasology' with security set to permissive and natives loading a second way (for KComputers)
+profile - Run 'Terasology' to play the game as a standard PC application (with Java FlightRecorder profiling)
+server - Starts a headless multiplayer server with data stored in [project-root]/terasology-server
+```
+
+To pass additional arguments to the PC Facade when running via Gradle, use the `--args` option (see the Gradle [Application Plugin](https://docs.gradle.org/6.9.1/userguide/application_plugin.html) documentation).
+For instance, to print the usage help, run
+
+```sh
+gradlew game --args='--help'
+```
+
+## Architecture
+
+The main entry point for the PC Facade is `Terasology#main`.
+Starting from there, the game engine is initialized and started according to the user inputs.
+
+<figure style="border:1px #cccccc solid; padding:4px;">
+  <!-- this image is an editable SVG created with Draw.io -->
+  <img 
+    style="display:block; margin-left:auto; margin-right:auto"
+    src="./docs/pc-facade-overview.drawio.svg" 
+    alt="Architectural overview of the PC Facade entry point."/>
+  <figcaption style="text-align:center;">Architectural overview of the PC Facade entry point.</figcaption>
+</figure>
```

Comment:
narg, fancy CSS does not work on Github :pensive: guess I'll use a good ol' `align="center"` somewhere then...

Reply:
```suggestion
<figure align="center">
  <!-- this image is an editable SVG created with Draw.io -->
  <img 
    src="./docs/pc-facade-overview.drawio.svg" 
    alt="Architectural overview of the PC Facade entry point."/>
  <figcaption>Architectural overview of the PC Facade entry point.</figcaption>
</figure>
```